### ISSUE TITLE
Move all existing va.gov-team content in final homes

### ISSUE THREAD

**Issue by rroueche** [department-of-veterans-affairs/va.gov-team#1241]
Once the new structure is in place, all files currently in va.gov-team repo should move moved into their correct homes. 

- [x] implement teams, product and platform folders - Done in Sprint 7
- [x] audit current va.gov-team files and determine new home - Done in Sprint 7
- [x] move files into new homes
- [x] identify where links need to be updated

**Comment by gunsch** [comment]
I ran [`markdown-link-check`](https://github.com/tcort/markdown-link-check) on the entire repo to look for broken links, now that we moved everything. `find . -name \*.md -exec markdown-link-check {} \; | tee ../markdown-link-check-output`

output: [markdown-link-check-output.txt](https://github.com/department-of-veterans-affairs/va.gov-team/files/3575935/markdown-link-check-output.txt)

**Comment by rroueche** [comment]
Creating another issue to correct the broken links w/the info Andrew shared.
